test/lean/registers.expected.lean

```diff
@@ -0,0 +1,16 @@
+import Out.Sail.Sail
+
+open Sail
+
+class MonadReg where
+  set_R0 : (BitVec 64) -> SailM Unit
+  get_R0 : SailM (BitVec 64)
+
```

Comment:
Probably more for the future than this pull request, but will this scale to one thousand registers?  (That's a ballpark figure for Arm-A 9.4.)